src/events/ready.ts

```diff
@@ -0,0 +1,15 @@
+import { Client, Events } from "discord.js";
+import Commands from "../Commands";
+
+export default (client: Client): void => {
+    client.on(Events.ClientReady, async() => {
+        if(!client.user || !client.application) {
+            return;
+        }
+
+        const commands = Commands.map(c => c.data);
+        await client.application.commands.set(commands);
```

Comment:
Maybe this should be guild specific, when developing due to global commands can take up to 1 hour to propogate it's changes.

> Guild commands are available only within the guild specified on creation. Guild commands update instantly. We recommend you use guild commands for quick testing, and global commands when they're ready for public use.

https://discord.com/developers/docs/interactions/application-commands#making-a-guild-command

Reply:
> Maybe this should be guild specific, when developing due to global commands can take up to 1 hour to propogate it's changes.
> 
> > Guild commands are available only within the guild specified on creation. Guild commands update instantly. We recommend you use guild commands for quick testing, and global commands when they're ready for public use.
> 
> https://discord.com/developers/docs/interactions/application-commands#making-a-guild-command

That's a great suggestion. Since we're only really expecting this bot to run on one guild, having the commands registered globally probably isn't necessary.

Resolved.